### PR TITLE
Refreshtoken and accesstoken

### PR DESCRIPTION
**Pull request by Enrikokokoko**

### CODE REVIEW

File: src/app.module.ts [hunk 1]
```diff
@@ -6,6 +6,14 @@ import { UserModule } from './user/user.module';
 import { DB_URL } from './config';
 
 @Module({
-  imports: [AuthModule, UserModule, MongooseModule.forRoot(DB_URL)],
+  imports: [
+    AuthModule, UserModule, MongooseModule.forRoot(DB_URL, {
+    connectionName:  'auth'
```
Comment (VladOSTAFI) [inline src/app.module.ts:11]: Why do you use `connectionName` in that case?

File: src/app.module.ts [hunk 2]
```diff
@@ -6,6 +6,14 @@ import { UserModule } from './user/user.module';
...
+  },
+  ),
+  MongooseModule.forRoot(DB_URL, {
```
Comment (VladOSTAFI) [inline src/app.module.ts:14]: Why do you create new connection to database here?

File: src/auth/auth.module.ts [hunk 3]
```diff
@@ -3,9 +3,21 @@ import { Module } from '@nestjs/common';
...
 @Module({
-  imports: [UserModule],
+  imports: [
+    UserModule,
+    MongooseModule.forFeature(
+      [{ name: User.name, schema: UserSchema }],
+      'auth'),
+      JwtModule.register({
+        global: true,
+        signOptions: { expiresIn: '60s' }
```
Comment (VladOSTAFI) [inline src/auth/auth.module.ts:18]: Please move '60s' to env variables

File: src/user/user.module.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ import { User, UserSchema } from './schemas/user.schema';
 
 @Module({
   imports: [
-    MongooseModule.forFeature([{ name: User.name, schema: UserSchema }]),
+    MongooseModule.forFeature([{ name: User.name, schema: UserSchema }], 'user'),
```
Comment (VladOSTAFI) [inline src/user/user.module.ts:10]: Why do you pass 'user' argument here?

**Review by VladOSTAFI — CHANGES_REQUESTED** [review]

File: src/app.module.ts (hunk 2 above)
Comment (Enrikokokoko) [reply src/app.module.ts:14]: service didn't start

File: src/user/user.module.ts (hunk 4 above)
Comment (Enrikokokoko) [reply src/user/user.module.ts:10]: service didn't start without argument 'user'

File: src/auth/auth.module.ts (hunk 3 above)
Comment (Enrikokokoko) [reply src/auth/auth.module.ts]: done

**Review by VladOSTAFI — APPROVED** [review]

**Pull request merged**